client/src/partials/vouchers/VoucherForm.service.js

```diff
@@ -163,27 +163,29 @@ function VoucherFormService(Vouchers, Constants, Session, VoucherItem, Cashboxes
     }
 
     // validate that this uses multiple accounts in the transaction
-    var hasUniqueAccounts = (uniqueAccountsArray.length > 1);
-    if (!hasUniqueAccounts) {
-      err = ERROR_SINGLE_ACCOUNT_TRANSACTION;
-    }
+    if(uniqueAccountsArray.length !== 0) {
```

Comment:
Actually... this could all be avoided by correcting the `hasUnqiueAccounts` typo.  The code should read:
```js
var hasUniqueAccounts = (uniqueAccountsArray.length !== 0);
```
instead of 
```
var hasUniqueAccounts = (uniqueAccountsArray.length > 1);
```

This is the typo causing this bug and you won't need to do the `if()`.